packages/suite/src/services/coingecko.ts

```diff
@@ -1,12 +1,56 @@
+/* eslint-disable max-classes-per-file */
 import { LastWeekRates, TickerId } from '@wallet-types/fiatRates';
 import FIAT_CONFIG from '@suite-config/fiat';
+import { differenceInMilliseconds } from 'date-fns';
 
+// const COINGECKO_API_BASE_URL = 'https://api.coingecko.com/api/v3';
```

Comment:
Is there any reason to keep this around? Let's either remove it altogether (still keeping it in git history) or make it a comment to make clear what is this URL.

```suggestion
// a proxy for https://api.coingecko.com/api/v3
```

Reply:
thx